src/components/TopComponent/TopComponent.js

```diff
@@ -0,0 +1,31 @@
+import React, {  PureComponent } from "react";
+import "./TopComponent.css";
+
+export class TopComponent extends PureComponent {
+  constructor(props){
+    super(props);
+    this.searchBar =React.createRef();
```

Comment:
now we can add ref via hook useRef pls read about it
https://reactjs.org/docs/hooks-reference.html#useref

Reply:
I will refer aliona , but hooks part is not covered yet.